app/src/main/java/com/framgia/soundclound/data/source/repository/GenresRepository.java

```diff
@@ -0,0 +1,52 @@
+package com.framgia.soundclound.data.source.repository;
+
+import android.support.annotation.StringDef;
+
+import com.framgia.soundclound.R;
+import com.framgia.soundclound.data.model.Genres;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.framgia.soundclound.data.source.repository.GenresRepository.ServiceName.*;
```

Comment:
WARNING: Using the '.*' form of import should be avoided - com.framgia.soundclound.data.source.repository.GenresRepository.ServiceName.*.

Reply:
check comment này đi em